static/style.css

```diff
@@ -0,0 +1,16 @@
+.button {
+    background-color: #000000;
+    border: none;
+    color: white;
+    padding: 20px 34px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 20px;
+    margin: 4px 2px;
+    cursor: pointer;
```

Comment:
Order alphabetically.

Reply:
done. is this standard for css?

Reply:
It's part of the Google HTML/CSS Style guide:
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order